src/masternodes/validation.cpp

```diff
@@ -2484,9 +2484,31 @@ static Res ProcessEVMQueue(const CBlock &block, const CBlockIndex *pindex, CCust
     res = cache.AddBalance(minerAddress, {DCT_ID{}, static_cast<CAmount>(blockResult.total_priority_fees)});
     if (!res) return res;
 
-    return Res::Ok();
+    return {blockResult.block_number, Res::Ok()};
+}
+
+
+static Res ProcessDST20Migration(const CBlock &block, const CBlockIndex *pindex, CCustomCSView &cache, const CChainParams& chainparams, const uint64_t evmQueueId, std::array<uint8_t, 20>& beneficiary, const bool evmEnabledOnBlockHead) {
```

Comment:
We do not need block, beneficiary or evmEnabledOnBlockHead passed to this function.